lookout/style/typos/preparation.py

```diff
@@ -200,9 +185,54 @@ def train_fasttext(data: pandas.DataFrame, params: Optional[Mapping[str, Any]] =
     model.save_model(params["fasttext_path"])
 
 
+def get_datasets(prepared_data: pandas.DataFrame, params: Optional[Mapping[str, Any]] = None,
+                 threads_number: int = multiprocessing.cpu_count(),
+                 ) -> Tuple[pandas.DataFrame, pandas.DataFrame]:
+    """
+    Create the train and the test datasets of typos.
+
+    1. Take the specified number of lines from the input dataset.
+    2. Make artificial typos in picked identifiers and split them into train and test.
+    3. Return results.
+    :param prepared_data: Dataframe of correct splitted identifiers. Must contain columns \
+                          Columns.Split, Columns.Frequency and Columns.Token.
+    :param params: Parameters for creating train and test datasets, options:
+                   train_size: Train dataset size.
+                   test_size: Test dataset size.
+                   typo_probability: Probability with which a token gets to be corrupted.
+                   add_typo_probability: Probability with which one more corruption happens to a \
+                                         corrupted token.
+                   train_path: Path to the .csv file where to save the train dataset.
+                   test_path: Path to the .csv file where to save the test dataset.
+    :param threads_number: Number of threads for multiprocessing.
```

Comment:
multiprocessing works wit htprocesses, not with threads. Those two are very different, especially for Python. So do you mean `processes_number` here?

Reply:
Yes, it's processes, changing it everywhere.